3135/2/user_info_class_set.rb

```diff
@@ -0,0 +1,160 @@
+# User class provides access to user info and manages immediate properties
+#:reek:TooManyInstanceVariables
+class User
```

Comment:
Name the file name as the class/module, but replacing CamelCase with snake_case.
Related to all other files